renku_notebooks/api/schemas.py

```diff
@@ -37,6 +40,16 @@ def validate_server_options(self, data, **kwargs):
             if server_options[option]["type"] == "boolean":
                 continue  # boolean options are already validated by marshmallow
             if data[option] not in server_options[option]["options"]:
+                # check if we allow arbitrary options
+                if server_options[option].get("allow_any_value"):
+                    value_range = server_options[option].get("value_range")
+                    if not value_range:
+                        raise RuntimeError("You must specify a value range.")
+                    if not _in_range(data[option], value_range):
+                        raise ValidationError(
+                            f"The value {data[option]} is outside of the range "
+                            f"{value_range.get('min')} to {value_range.get('max')}."
+                        )
```

Comment:
```suggestion
                        )
                    return
```

Reply:
I think I got the indent right. Without this return statement the code continues to hit the validation error below even thought it should not.

Reply:
that should actually be `continue` - my bad